authors/apps/comments/views.py

```diff
@@ -86,7 +92,7 @@ def get(self, request, pk):
         serialized_data = self.serializer_class(self.comment)
         return Response({"comment": serialized_data.data,
                          "message": (
-                    "Successfully returned details of comment: {}".format(pk)
+                             f"Successfully returned details of comment: {pk}"
```

Comment:
Is the pk here same as the id? If that's the case then this return is going to be e.g "Successfully returned details of comment: 1" Could you do without the pk if that's the case, or could you return the comment body

Reply:
I think we'd need a little clarification from @pbkabali on this one, my change was a basically a reformat of already existing code.

Reply:
@missvicki and guys, I made sure the message is descriptive enough for whoever is going to consume the api. In my view, there is no problem with it.

Reply:
@missvicki does this work for you?

Reply:
Still not convinced, but we can roll with it based on the discussion we had.